packages/cozy-intent/src/api/services/NativeMessenger.ts

```diff
@@ -19,8 +19,12 @@ export class NativeMessenger implements Messenger {
   }
 
   public postMessage = (message: Record<string, unknown>): void => {
-    const script = `window.postMessage(${JSON.stringify(message)})`
-    this.injectJavaScript?.(script)
+    try {
+      const script = `window.postMessage(${JSON.stringify(message)})`
+      this.injectJavaScript?.(script)
+    } catch (error) {
+      log(strings.noWebviewFound)
```

Comment:
Only happens when going back to home as of now, no breaking impact.
Post-me tries to respond to backToHome call with the cozyapp webview, but that webview doesn't exist anymore when it does that